### ISSUE TITLE
can not find <hdf5.h> when use VS2015 X64

### ISSUE THREAD

**Issue by HolaMirai** [mariusmuja/flann#314]
HELLO.
I have compiled this code sucessfully, but when i use the compiled code in VS2015 and x64 platform , errors occured. my test code is 
`
#include <flann/flann.hpp>
#include <flann/io/hdf5.h>
#include <stdio.h>
int main(int argc, char** argv)
{
	int nn = 3;
	flann::Matrix<float> dataset;
	flann::Matrix<float> query;
	flann::load_from_file(dataset, "dataset.hdf5", "dataset");
	flann::load_from_file(query, "dataset.hdf5", "query");
	flann::Matrix<int> indices(new int[query.rows*nn], query.rows, nn);
	flann::Matrix<float> dists(new float[query.rows*nn], query.rows, nn);
	// construct an randomized kd-tree index using 4 kd-trees
	flann::Index<flann::L2<float> > index(dataset, flann::KDTreeIndexParams(4));
	index.buildIndex();
	// do a knn search, using 128 checks
	index.knnSearch(query, indices, dists, nn, flann::SearchParams(128));
	flann::save_to_file(indices, "result.hdf5", "result");
	
	delete[] dataset.ptr();
	delete[] query.ptr();
	delete[] indices.ptr();
	delete[] dists.ptr();
	return 0;
}
`
the Error messages are:
1. can`t open the file "hdf5.h"
2. int the file<hdf5.h> of line 50 : "hit_t"   has no storage class or type specifier.
![image](https://cloud.githubusercontent.com/assets/13817461/21283286/ff914f4c-c43a-11e6-9d3c-152d0f2cacda.png)

do anyone know how can i fix it ？thank you in advance.

**Comment by HolaMirai** [comment]
yes i found what`s wrong with my job, it is because when i cmake the project, i did not set the path of HDF5,thus it can`t find hdf5.h.

**Comment by ShuMeng** [comment]
Dear HolaMirai,

Hope your problem is solved. 
I am having the same problem with you.  I am very new to c++, could you please show me how to connect HDF5 to my current project?
Thank you soooo much in advance.

Sussi
